cmd/lncli/walletrpc_active.go

```diff
@@ -1193,6 +1195,191 @@ func listAddresses(ctx *cli.Context) error {
 	return nil
 }
 
+var signMessageWithAddrCommand = cli.Command{
+	Name:  "signmessage",
+	Usage: "Sign a message with the private key of the provided address",
+	Description: `
```

Comment:
Need to add `Category` here.

Reply:
hm. not sure, the higher-level command wallet has the category `wallet` it the new commands are part of wallet => address so I guess we are good without a category here